proto/tendermint/statesync/types.proto

```diff
@@ -46,4 +50,13 @@ message LightBlockRequest {
 
 message LightBlockResponse {
   tendermint.types.LightBlock light_block = 1;
+}
+
+message ParamsRequest {
+  uint64 height = 1;
+}
+
+message ParamsResponse {
+  uint64                           height           = 1;
+  tendermint.types.ConsensusParams consensus_params = 2 [(gogoproto.nullable) = false];
 }
```

Comment:
Is this field mean to represent the height at which those params apply? I.e. if this field is 13 that means that these were the params at height 13? If that's the case, I think that this field would be helpful for debugging so that clients can easily tell what height they are receiving a response for.

Reply:
> Is this field mean to represent the height at which those params apply?

Exactly, but even without this field, the state provider knows which height it is expecting the consensus params from the peer to be so it should still be able to log it.